test/gtest/ucm/malloc_hook.cc

```diff
@@ -387,3 +387,43 @@ UCS_TEST_F(malloc_hook_cplusplus, new_delete) {
     ucm_unset_event_handler(UCM_EVENT_VM_UNMAPPED, mem_event_callback,
                             reinterpret_cast<void*>(this));
 }
+
+extern "C" { int ucm_dlmallopt_get(int); };
+UCS_TEST_F(malloc_hook_cplusplus, mallopt) {
+
+    int v;
+    char *p;
+
+    /* This test can not be run with the other
+     * tests because it assumes that malloc hooks
+     * are not initialized
+     */
+    if (getenv("UCX_GTEST_MALLOPT") == NULL) {
```

Comment:
i would set MALLOC_TRIM_THRESHOLD_ in test_jenkins.sh, and inside this test - check if MALLOC_TRIM_THRESHOLD_ is present in getenv(). This way would not need to define another env var and no need to disable rcache.

Reply:
you still have to disable rcache because it triggers malloc initialization way before the actual test

Reply:
ok, but still can avoid UCX_GTEST_MALLOPT